x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/action_form.tsx

```diff
@@ -141,15 +141,22 @@ export const ActionForm = ({
       });
     }
   }
+  const preconfiguredMessage = i18n.translate(
```

Comment:
It would be nice to have a space between the connector name and `preconfiguredMessage`. Right now it would show as `Foo(pre-configured)` and with a space it would be `Foo (pre-configured)`